web/app/js/components/ServiceMesh.jsx

```diff
@@ -109,6 +118,15 @@ class ServiceMesh extends React.Component {
     ];
   }
 
+  getInstalledExtensions() {
+    const { extensions } = this.state;
+    let extensionList = [];
+    if (!_isEmpty(extensions.extensions)) {
+      extensionList = extensions.extensions.map(extension => extension.metadata);
+    }
+    return extensionList;
```

Comment:
I think it'd be easier here to just return the resulting array from map.

We declare an empty array on L123, if the state is not empty (and we have a list of extensions) then we map through and change an extension to be its associated metadata. If the array is empty then mapping through it will still give an empty array, right?

```suggestion
    const { extensions } = this.state;
    return extensions.extensions.map(extension => extension.metadata);
```

also, any way we can avoid doing `extensions.extensions` and instead do just `extensions.map`?

Reply:
So if the thing is, the test passes a different value to the service mesh component on component did mount. I noticed it was passing 

```
{ metrics: [] }
```

So the `extensions.extensions` resulted in undefined. And that's why the test was failing initially. I had to put a check to ensure it exists before mapping it

Reply:
For the wrapped heading - would check it out again and see why its like that